visualdl/logic/pybind.cc

```diff
@@ -74,14 +74,17 @@ PYBIND11_MODULE(core, m) {
       #undef READER_ADD_HISTOGRAM
 
       // clang-format on
-      .def("get_image",
-           [](vs::LogReader& self, const std::string& tag) {
-             auto tablet = self.tablet(tag);
-             return vs::components::ImageReader(self.mode(), tablet);
-           })
+      .def("get_image", [](vs::LogReader& self, const std::string& tag) {
```

Comment:
these seem just some code style formatting, and we each other have a different clang-format config that results in some diff, maybe we need a unified version of config that makes we have the same code style?

In paddle, the clang-format 3.8 and google c++ style is used, different config and version may lead to some diff.

We can reference [paddle's .clang-format configuration](https://github.com/PaddlePaddle/Paddle/blob/develop/.clang-format)

Reply:
yes, I just updated clang-format